tests/commands/test_bump_command.py

```diff
@@ -262,3 +266,38 @@ def test_bump_in_non_git_project(tmpdir, config, mocker):
         with pytest.raises(NotAGitProjectError):
             with pytest.raises(ExpectedExit):
                 cli.main()
+
+
+def test_none_increment_exit_should_be_a_class():
+    assert inspect.isclass(NoneIncrementExit)
+
+
+def test_none_increment_exit_should_be_expected_exit_subclass():
+    assert issubclass(NoneIncrementExit, ExpectedExit)
+
+
+def test_none_increment_exit_should_exist_in_bump():
+    assert hasattr(bump, "NoneIncrementExit")
+
+
+def test_none_increment_exit_is_exception():
+    assert bump.NoneIncrementExit == NoneIncrementExit
+
+
+@pytest.mark.usefixtures("tmp_commitizen_project")
+def test_none_increment_should_not_call_git_tag(mocker, tmp_commitizen_project):
```

Comment:
Can u explain me what's going on in this test?
we are creating a tag, we bump and because there's no increment, it will rise an exception?
What is `assert_not_called` doing?

Reply:
Helo @Woile , thanks for the review.

As I usually use TDD, I write tests that break and write code to pass the test and refactor.

`assert_not_called` asserts that `git.tag` was not called, I started here: if there is no version increment, git.tag should not be called. So if by any case git.tag is called when there is no version increment, test will break.

But if git.tag does not respond properly, I cannot perform test that breaks. So I created a dummy return value here:
    dummy_value = git.tag("0.0.2")
    git.tag = MagicMock(return_value=dummy_value)

So when when I run a bump with None increment (i.e.: with a `test: ...` commit), when I run `cz bump --yes` git.tag should not be called and raise `NoneIncrementExit`)

Let me know your thoughts.